pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -393,6 +407,7 @@ func (qt *QueueTracker) canRunApp(hierarchy []string, applicationID string, trac
 // canBeRemoved Start from root and reach all levels of queue hierarchy to confirm whether corresponding queue tracker
 // object can be removed from ugm or not. Based on running applications, resource usage, child queue trackers, max running apps, max resources etc
 // it decides the removal. It returns false the moment it sees any unexpected values for any queue in any levels.
+// Note: Lock free call. The RLock of the linked tracker(UserTracker/GroupTracker) should be held before calling this function.
```

Comment:
`tracker(UserTracker/GroupTracker)` => `tracker (UserTracker and GroupTracker)`

Reply:
fixed.